test/export/test_torchbind.py

```diff
@@ -1301,7 +1301,7 @@ def f(tq, x):
         )
         self.assertEqual(cnt.frame_count, 4)
 
-    @parametrize("backend", ["eager", "aot_eager"])
+    @parametrize("backend", ["eager", "aot_eager", "inductor"])
```

Comment:
> Currently the call_torchbind ops get directly inlined into the python code. I originally was thinking we could codegen call_torchbind calls to be object.method, but it seems like directly inlining the call_torchbind calls work :thonk:

Inline sounds fine to me. We've had precedence for this (calling a HOP in the inductor graph) before; auto_functionalize used to be inlined into the graph before being reinplaced